### PR TITLE
Created the plotfasta function and loadkmers function

### PR DESCRIPTION
**Pull request by Thohil**
Created a function to plot fasta data using the kmers
Lazar created a function to load in kmers from the .model file

### CODE REVIEW

File: Project.toml
Comment (JokingHero) [inline Project.toml]: PyPlot not needed

File: src/NanoporeMultiEventalign.jl
Comment (JokingHero) [inline src/NanoporeMultiEventalign.jl]: this should be at the top

File: src/NanoporeMultiEventalign.jl
Comment (JokingHero) [inline src/NanoporeMultiEventalign.jl]: should not be here

File: src/persistence.jl
Comment (JokingHero) [inline src/persistence.jl]: you can put this into documentation with 

"

"

File: src/persistence.jl
Comment (JokingHero) [inline src/persistence.jl]: should be parameter with default

File: src/utils.jl
Comment (JokingHero) [inline src/utils.jl]: keep the naming conventions consistent with julia style

File: src/utils.jl
Comment (JokingHero) [inline src/utils.jl]: assumes the size of the kmer

File: src/utils.jl
Comment (JokingHero) [inline src/utils.jl]: keep spaces etc

File: src/utils.jl
Comment (JokingHero) [inline src/utils.jl]: plot path should be input

File: src/utils.jl
Comment (JokingHero) [inline src/utils.jl]: instead make funciton that processes fasta into kmer mean points

**Review by JokingHero — CHANGES_REQUESTED** [review]

File: src/NanoporeMultiEventalign.jl
Comment (JokingHero) [inline src/NanoporeMultiEventalign.jl]: should be values?
fasta_to_model maybe? 

After you fix this try merging this branch to the main, fix conflicts beforehand.

**Review by JokingHero — APPROVED** [review]
Adress my comment and merge this to the main.

**Comment by Thohil** [comment]
It says "Only those with write access to this repository can merge pull requests." so i am not sure if i have permission to merge the changes into main
image

**Comment by JokingHero** [comment]
You have not accepted the invitation to be the contributor, its waiting for some time now.

**Comment by Thohil** [comment]
I found the invitation, however github invitations expire after 7 days so i can't use it. is it possible for you to resend the invitation?

**Pull request merged**
